client/src/js/modules/dc/views/attachments.js

```diff
@@ -7,6 +7,44 @@ define(['marionette',
     'utils'], 
     function(Marionette, Backgrid, TableView, attachments, LogView, CloudUploadView, utils) {
 
+    var PathCell = Backgrid.Cell.extend({
+        events: {
+            'click .copy-path': 'copyPathToClipboard'
+        },
+
+        render: function() {
+            var filePath = this.model.get('FILEPATH')
+            var displayedPath = filePath.split('/').slice(0,4).join('/')+'/...'
```

Comment:
Nitpick, but if the full path is less than 4 levels deep, this would display ellipsis regardless, it seems

Does that seem like a problem?

Reply:
Probably not a problem, filepaths are always going to start /dls/beamline/data/...,, but you're correct and it's an easy fix so I've added something to cope with that.